packages/mdl-ripple/index.js

```diff
@@ -32,6 +32,23 @@ export default class MDLRipple extends MDLComponent {
     return ripple;
   }
 
+  static createAdapter(instance) {
+    return {
+      browserSupportsCssVars: () => supportsCssVariables(window),
```

Comment:
I see this was just a move, but to satisfy my curiosity:
Under what circumstances would a component using `MDLRippleFoundation` want to override this?